test/js/utils/CanvasUtilsTest.js

```diff
@@ -0,0 +1,15 @@
+describe("Canvas utils", function() {
+
+  it("returns a single blank frame when provided a blank spritesheet set to ignore blank frames", function() {
+    var width = 10;
+    var height = 10;
+    var blankData = pskl.utils.CanvasUtils.createCanvas(width, height).toDataURL();
+    var blankImg = new Image(width, height);
+    blankImg.src = blankData;
+
+    var blankSpritesheetFrames = pskl.utils.CanvasUtils.createFramesFromImage(blankImg, 0, 0, width, height, true, true);
+
+    expect(blankSpritesheetFrames.length).toBe(1);
+    expect(blankSpritesheetFrames[0].toDataURL()).toBe(blankData);
+  });
```

Comment:
Excellent!

I don't think we should try to add full coverage for this method right  now, but there's at least one more behavior _we_ added that I think we should check:

> Given we have an empty spritesheet of many frames
> When we createFramesFromImage
> Then we should only get one frame back

I think that'd look something like this:
```js
it("returns only one blank frame when given many but told to ignore blanks", function() {
  var imageSize = 50;
  var frameSize = 10;
  var blankData = pskl.utils.CanvasUtils.createCanvas(imageSize, imageSize).toDataURL();
  var blankImg = new Image(imageSize, imageSize);
  blankImg.src = blankData;
  var blankSpritesheetFrames = pskl.utils.CanvasUtils.createFramesFromImage(blankImg, 0, 0, frameSize, frameSize, true, true);

  expect(blankSpritesheetFrames.length).toBe(1);
  expect(blankSpritesheetFrames[0].width).toBe(frameSize);
  expect(blankSpritesheetFrames[0].height).toBe(frameSize);
});
```